packages/discord.js/typings/index.d.ts

```diff
@@ -1008,7 +1016,7 @@ export class Client<Ready extends boolean = boolean> extends BaseClient {
   ): this;
   public on<Event extends string | symbol>(
     event: Exclude<Event, keyof ClientEvents>,
-    listener: (...args: any[]) => Awaitable<void>,
+    listener: (...args: readonly any[]) => Awaitable<void>,
```

Comment:
Can you check this one specific change won't break anything? It also diverts from the types in node and AEE

Reply:
I'll just make an exception for `on` and `once` methods as to keep it the same as node's types